scaleway/resource_token.go

```diff
@@ -27,6 +27,21 @@ func resourceScalewayToken() *schema.Resource {
 				Computed:    true,
 				Description: "The userid of the associated user.",
 			},
+			"access_key": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The access_key.",
+			},
+			"secret_key": {
+				Type:        schema.TypeString,
+				Computed:    true,
```

Comment:
we probably want to set `Sensitive: true` here?

Reply:
done :)